js/template.js

```diff
@@ -695,6 +695,7 @@ jQuery(document).on('bootstrap3:page-icons', function(event) {
     'pinterest'   : (function(){ return [ 'https://pinterest.com/pin/create/button/?url=', url, '&description=', title ].join(''); })(),
     'telegram'    : (function(){ return [ 'https://telegram.me/share/url?url=', url ].join(''); })(),
     'whatsapp'    : (function(){ return [ 'whatsapp://send?text=', title, ': ', url ].join(''); })(),
+    'yammer'      : (function(){ return [ 'https://www.yammer.com/ramboll.com/messages/new?login=true&trk_event=yammer_share&status=', url, '#/Messages/bookmarklet'].join(''); })(),
```

Comment:
The corrected url is `https://www.yammer.com/messages/new?login=true&trk_event=yammer_share&status=`. After the login you will be redirected into your company profile.

Reply:
Slight oversight on my part, thanks!